plugins/kubernetes/app/views/samson_kubernetes/_build_show.erb

```diff
@@ -0,0 +1,4 @@
+<div class="form-group">
+  <%= check_box_tag :kubernetes_job%>
+  <%= form.label :kubernetes_job, "Use Remote Kubernetes Job", class: "control-label" %>
```

Comment:
should this be a column instead ? ... then we don't need extra_build_params ... just build_params and let it persist ... and no additional UI for build/show ...